setup.py

```diff
@@ -14,10 +14,19 @@
 
 import io
 import os
+import platform
 
 import setuptools
 
 
+grpc_version = "grpcio >= 1.44.0, < 2.0dev"
+
+# Override version to 1.44 for M1
+# https://github.com/googleapis/python-pubsub/issues/609
+if platform.system() == "Darwin" and platform.machine() == "arm64":
+    "grpc == 1.44.0"
```

Comment:
Is there a reason to pin to 1.44.0 exactly and not >=1.44 for M1?

Reply:
@galuszkak Friendly ping, want to make sure we can get you this fix soon.

Reply:
They have a new grpc version, that might be interesting for this 

https://github.com/grpc/grpc/issues/25082#issuecomment-1439223987

Reply:
@galuszkak I'm going to close this PR, since there's a fix in: [1.51.3](https://pypi.org/project/grpcio/1.51.3/)